native/src/components/Header.tsx

```diff
@@ -145,7 +147,10 @@ const Header = (props: PropsType): ReactElement => {
     }
 
     const description = routeCityModel.prefix ? ` (${routeCityModel.prefix})` : ''
-    return `${routeCityModel.sortingName}${description}`
+    const cityNameLength = routeCityModel.sortingName.length
+    return cityNameLength < dimensions.deviceWidth / dimensions.headerTextSize
```

Comment:
I'm not sure whether I understand this properly:

If the length of the sorting name is longer than the amount of characters we can fit on the screen, then we add a newline.

I see a few issues here:
* Sorting names can be anything, we do not have rules for them
* A character with fontsize 20 is not 20 pixel wide. This would be only an approximitation


Can we somehow let React Native handle the text wraping/breaking?

Reply:
Indeed its only an approximation. All i found for react native font scaling was using something like 'vw' and this can be achieved with device width and a scaling factor which is 5% by default. According to some spacing issues i decreased it to 4%
there are some libs doing simular things:
https://www.npmjs.com/package/react-native-responsive-fontsize

React Native deals by default with line breaking but not names including "-", thats why i'm replacing it to force breaking.

Reply:
K, yeah hacky but its probably the best we can hope for :D